pkg/sypgp/sypgp.go

```diff
@@ -105,71 +113,89 @@ func PublicPath() string {
 	return filepath.Join(DirPath(), "pgp-public")
 }
 
-// PathsCheck creates the sypgp home folder, secret and public keyring files
-func PathsCheck() error {
-	// create the sypgp base directory
-	if err := os.MkdirAll(DirPath(), 0700); err != nil {
+func ensureDirPrivate(dn string) error {
```

Comment:
Same here: it seems to me that this is something that we can use in many different places throughout the code so it might be convenient to put it in a utility package? Maybe in `internal/pkg/util/fs/helper.go`?

Reply:
Added TODOs for this.